README.md

```diff
@@ -29,102 +39,26 @@ This project has 2 parts: the Docker environment and a set of tools for theme de
 ```
 # copy the files
 git clone https://github.com/visiblevc/wordpress-starter.git
-rm -rf .git Dockerfile run.sh README.md CHANGELOG.md ISSUE_TEMPLATE.md
+
+# navigate to example directory
+cd wordpress-starter/example
 
 # start the website at localhost:8080
-docker-compose up
+docker-compose up -d && docker-compose logs -f wordpress
 ```
 
 **NOTE:** If you run on MacOS with Docker in VirtualBox, you will want to forward the port by running this `VBoxManage controlvm vm-name natpf1 "tcp8080,tcp,127.0.0.1,8080,,8080"`. If you use another port than `8080`, change it in the command.
 
-##### This repository does 2 things:
-
-1. Include the files to create a wordpress Docker image (visiblevc/wordpress)
-2. Include build tools to develop wordpress themes (gulp)
-
-If you don't plan to build the Docker image yourself, you shouldn't care for 1. We publish the image on Docker Hub and you can grab it directly from there. That's why you can safely remove the Dockerfile and run.sh.
-
-The reason we remove `.git`, `README.md` and `CHANGELOG.md` is because we assume you will start your own repository, named after your project. There is virtually no benefit keeping ties with our remote git repository.
-
 ### Available Images
 
 | PHP Version | Tags |
 | ----------- | ---- |
-| **7.0**     | `latest` `latest-php7.0` `<version>-php7.0` |
+| **7.1**     | `latest` `latest-php7.1` `<version>-php7.1` |
+| **7.0**     | `latest-php7.0` `<version>-php7.0` |
 | **5.6**     | `latest-php5.6` `<version>-php5.6`|
 
 If you need a specific version, look at the [Changelog](CHANGELOG.md)
 
-### Introduction
-
-We wrote a series of articles explaining in depth the philosophy behind this project:
-
-- [Intro: A slightly less shitty WordPress developer workflow](https://visible.vc/engineering/wordpress-developer-workflow/)
-- [Part 1: Setup a local development environment for WordPress with Docker](https://visible.vc/engineering/docker-environment-for-wordpress/)
-- [Part 2: Setup an asset pipeline for WordPress theme development](https://visible.vc/engineering/asset-pipeline-for-wordpress-theme-development/)
-- [Part 3: Optimize your wordpress theme assets and deploy to S3](https://visible.vc/engineering/optimize-wordpress-theme-assets-and-deploy-to-s3-cloudfront/)
-- Part 4: Auto deploy your site on your server (coming)
-
-### Example
```

Comment:
My question wasn't clear. I think that there is value in the documentation. You might want to use this without relying on the example at all no?